docs/News.rst

```diff
@@ -0,0 +1,59 @@
+.. _News:
```

Comment:
This file must be removed no ?

Reply:
I removed the file in my desktop's folder which points to the preprod.